.github/workflows/build.yml

```diff
@@ -80,7 +80,7 @@ jobs:
       - name: Set up Node.js
         uses: actions/setup-node@v6
         with:
-          node-version: lts/*
+          node-version: 24.x
```

Comment:
FWIW `lts/*` is intentional – we used to have to maintain this line across many GHA build files, which was a bit of a nuisance!